2020q1/manpages.md

```diff
@@ -0,0 +1,9 @@
+## FreeBSD Manpages overhaul ##
+
+Contact: Gordon Bergling <gbergling@gmail.com> 
+
+I am currently working on an overhaul for the FreeBSD manpages by updating the HISTORY and STANDARDS sections and while here creating new manpages for parts of the system that missing documentation. FreeBSD has already one of the best documentation available for an UNIX-like operation system, but there are parts that could be improved.
+
+For the parts that have been already improved you can have a look at my Phabricator account at https://reviews.freebsd.org/.
```

Comment:
You probably want directly link to your profile: https://reviews.freebsd.org/p/gbergling_gmail.com/

Reply:
I changed the link, thank you for the hint.